eden/RNA.py

```diff
@@ -179,8 +180,70 @@ def _align_sequence_structure(self, seq, neighs):
                 clean_seq, clean_struct = rnafold.RNAfold_wrapper(seq[1])
             else:
                 clean_seq, clean_struct = make_seq_struct(seed, struct)
+            if structure_deletions:
+                clean_struct = self._clean_structure(clean_seq, clean_struct)
+
         return header, clean_seq, clean_struct, energy
 
+    def _clean_structure(self, seq, stru):
+        '''
+        Parameters
+        ----------
+        seq : basestring
+            rna sequence
+        stru : basestring
+            dotbracket string
+
+        Returns
+        -------
+        the structure given may not respect deletions in the sequence.
```

Comment:
This is more a general description of the function not what you would expect as a return statement :)